action-server/covidflow/actions/question_answering_form.py

```diff
@@ -58,6 +59,26 @@ async def _activate_if_required(
             dispatcher.utter_message(template="utter_can_help_with_questions")
             dispatcher.utter_message(template="utter_qa_disclaimer")
 
+            responses = domain.get("responses", {})
+            qa_samples_categories = [
+                key for key in responses.keys() if key.startswith("utter_qa_sample_")
+            ]
+            random_qa_samples_categories = random.sample(
+                qa_samples_categories, k=min(len(qa_samples_categories), 3)
+            )
+
+            random_qa_samples = [
+                f"- {random.choice(value).get('text')}"
+                for key, value in responses.items()
+                if key in random_qa_samples_categories
+            ]
+
+            if len(random_qa_samples) > 0:
+                dispatcher.utter_message(
+                    template="utter_qa_sample",
+                    sample_questions="\n".join(random_qa_samples),
```

Comment:
Maybe join with `\n- `? It would be clearer that these are distinct examples if they appear in a list

Reply:
They are in a list, look above :slightly_smiling_face:

Reply:
Ah! It's so subtle I hadn't seen it.